### PR TITLE
remove ancient/unused scripts

### PR DESCRIPTION
**Pull request by jhoblitt**
Removes ancient/unused/broken scripts, all installation of files other then `./ups`, a noop "doc build" and uneeded `scons` machinery.  This converts this product essential into a table file for declaring variables and the SCM home for `newinstall.sh`.

### CODE REVIEW

**Comment by jhoblitt** [comment]
This PR cuts a bit deeper than #38 but I believe all of the removed scripts are [long] out of disuse.

**Comment by RobertLuptonTheGood** [comment]
Rather than remove the autoconf check for pre-requisites, wouldn't it have been better to fix it?

File: ups/lsst.table [hunk 1]
```diff
@@ -1,7 +1,6 @@
 setupRequired(scons)
 setupRequired(sconsUtils)
 
-envPrepend(PATH, ${PRODUCT_DIR}/bin)
 envPrepend(EUPS_PKGROOT, http://sw.lsstcorp.org/eupspkg, |)
 
 # Deprecated: here for backwards compatibility, but discouraged from using
```
Comment (timj) [inline ups/lsst.table]: So what about these deprecated entries?

Comment (jhoblitt) [inline ups/lsst.table]: I'm fairly confident that `LSST_HOME` from this package isn't being relied on but I'm not sure.  Leaving it doesn't seem to hurt for the moment but I would certainly like to get rid of it.

**Comment by jhoblitt** [comment]
@RobertLuptonTheGood The autoconf check isn't being used and is out of date, so we've obviously been living with out it.  This package isn't at the bottom of a deep chain so I'm not sure how much good it would do anyways. If it was, it would almost certainly be problematic for products such as `qserv` as it would force unneeded dependencies. IMHO - probing the system for deps should be done on a per package basis where there is some hope of the probes being kept up to date.

**Pull request merged**